### PR TITLE
feature : TX 2412 upgrade (TX EDC 0.8.0 release)

### LINKED ISSUE #35: Manually triggering the "Chart testing" workflow does skip the test
The current [chart testing workflow](https://github.com/eclipse-tractusx/knowledge-agents-edc/blob/main/.github/workflows/helm-chart-lint.yml#L85) does skip testing, if there are no changes to the Chart.
When triggering the workflow manually, then there are usually no changes, since the release was done before.

There is a flag `--charts` described in the [chart testing docs](https://github.com/helm/chart-testing/blob/main/doc/ct_install.md), that could be used to explicitly specify the charts to test. Using this flag, skips the change detection

### PR DESCRIPTION
**Pull request by drcgjung**
## WHAT

Adapts dependencies and base images to the TX EDC 0.8.0 release. 

## WHY

Provide the most possible compatibility of the agent planes.

## FURTHER NOTES

Closes #35 # <-- _insert Issue number if one exists_

### CODE REVIEW

**Comment by drcgjung** [comment]
there is a "restricted" dependency 
maven/mavencentral/org.flywaydb/flyway-database-postgresql/10.21.0, , restricted, clearlydefined
which comes through TX EDC ... an IP check should already been issued at eclipse, I would guess.

File: DEPENDENCIES
Comment (almadigabor) [inline DEPENDENCIES:1]: Please update the DEPENDENCIES file before I merge the PR. There should be no restricted libraries once you run the dash tool.

**Review by almadigabor — CHANGES_REQUESTED** [review]
Just one minor request.

File: DEPENDENCIES
Comment (drcgjung) [reply DEPENDENCIES:1]: There are clear rules for commiters and there are the obvious exceptions

https://github.com/eclipse-tractusx/tractusx-edc/blob/25da0bc59107d8bc379d8162002698165ecefff9/DEPENDENCIES#L571

So If you could please open an IP check on that one line

maven/mavencentral/org.flywaydb/flyway-database-postgresql/10.21.0, , restricted, clearlydefined

Comment (almadigabor) [reply DEPENDENCIES:1]: I know the process, might have been on a different branch when I ran the command. Now the IP ticket is created, let's wait for approval then the DEPENDENCIES file can be updated.

Output:
```
A review is required for maven/mavencentral/org.flywaydb/flyway-database-postgresql/10.21.0.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/17433 .
```

**Pull request merged**
